### PR TITLE
Cl 6 - fix core-lib creation command line

### PR DESCRIPTION
**Pull request by oren-una**
generate command line seems to work well

### CODE REVIEW

File: core_lib_generator/config_collectors/database.py [hunk 1]
```diff
@@ -85,9 +85,14 @@ def _generate_db_config(
     migrate: bool = False
 ) -> dict:
     config = _build_url(db_type, db_name, db_username, db_password, db_port, db_host)
+    if db_type:
+        db_type: str = 'core_lib.connection'
+        if migrate:
+            db_type += '.sql_alchemy_connection_registry.SqlAlchemyConnectionRegistry'
```
Comment (shubham-shay-te) [inline core_lib_generator/config_collectors/database.py:91]: if the migrate is `False` the db_type string will just be `'core_lib.connection'`, talk with me about it

Comment (shubham-shay-te) [inline core_lib_generator/config_collectors/database.py:91]: if the migrate is `False` the db_type string will just be `'core_lib.connection'`, talk with me about it

**Review by shubham-shay-te — CHANGES_REQUESTED** [review]

File: core_lib_generator/config_collectors/database.py [hunk 2]
```diff
@@ -85,9 +85,12 @@ def _generate_db_config(
     migrate: bool = False
 ) -> dict:
     config = _build_url(db_type, db_name, db_username, db_password, db_port, db_host)
+    if db_type:
+        db_type: str = 'core_lib.connection.sql_alchemy_connection_registry.SqlAlchemyConnectionRegistry'
     return {
         'env': config['env'],
         'connection': {
+            'type': db_type,
```
Comment (shubham-shay-te) [inline core_lib_generator/config_collectors/database.py:93]: is this being used by the generator?

**Review by shubham-shay-te — CHANGES_REQUESTED** [review]

**Review by shubham-shay-te — APPROVED** [review]

File: core_lib_generator/config_collectors/database.py (hunk 2 above)
Comment (oren-una) [reply core_lib_generator/config_collectors/database.py:93]: The generator creates this line and looks for the 'type' key while making the yaml file

**Pull request merged**